src/main/resources/public/jsx/contact.jsx.js

```diff
@@ -81,9 +84,32 @@ var ContactForm = React.createClass({
                 subject: '',
                 body: '',
                 copyToSender: false
-            }
+            },
+            captchaToken: ''
         });
     },
+    componentDidMount() {
+        this.getCaptchaSiteKey();
+    },
+    getCaptchaSiteKey() {
+        $.ajax({
+            url: contact_service + '/captchasitekey',
+            type: 'GET',
+            contentType: 'application/json',
+            success: function (data) {
+                this.setState({ sitekey: data });
+            }.bind(this),
```

Comment:
does'nt work without `bind()`, I have this error :
```
contact.jsx.js?c946:100 Uncaught TypeError: this.setState is not a function
``` 
And we can't add constructor to bind `getCaptchaSiteKey()` 
```
Uncaught Error: ReactClassInterface: You are attempting to define `constructor` on your component more than once. This conflict may be due to a mixin.
```